### PR TITLE
Feature/oe 5791

### PR DESCRIPTION
**Pull request by bukamanush**

### CODE REVIEW

File: migrations/m151015_123216_remove_lens_foreign_keys.php [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+	/*
+	 *
+	 */
+	public function up()
+	{
+		//Removing the foreign key for saving data without any lense.
+		$this->dropForeignKey('ophinbiometry_lenstype_lens_l_fk', 'et_ophinbiometry_selection');
+		$this->dropForeignKey('ophinbiometry_lenstype_lens_r_fk', 'et_ophinbiometry_selection');
+
+		$this->delete('ophinbiometry_lenstype_lens',"name = 'unknown'");
```
Comment (himanshupatel79) [inline migrations/m151015_123216_remove_lens_foreign_keys.php:15]: Remove this line.

File: views/default/form_Element_OphInBiometry_Selection_fields.php [hunk 2]
```diff
@@ -2,7 +2,15 @@
 <div class="element-fields">
 	<div class="row">
 		<div class="large-12 column">
-			<?php echo $form->dropDownList($element, 'lens_id_'.$side, CHtml::listData(OphInBiometry_LensType_Lens::model()->activeOrPk($element->{'lens_id_'.$side})->findAll(array('order'=> 'display_order asc')),'id','name'),array('empty'=>'- Please select -'),null,array('label'=>3, 'field'=>6))?>
+			<?php
+			$post = Yii::app()->request->getPost('Element_OphInBiometry_Selection');
```
Comment (himanshupatel79) [inline views/default/form_Element_OphInBiometry_Selection_fields.php:6]: 1) Business Logic should be in Controller, not in View. 
2) Remove this code base, As it is not required.  It insert default value 0 in db , if lens is not selected .

File: migrations/m151015_123216_remove_lens_foreign_keys.php (hunk 1 above)
Comment (vetusko) [inline migrations/m151015_123216_remove_lens_foreign_keys.php:15]: We need to keep this line to revert the changes made by the down migration

File: views/default/form_Element_OphInBiometry_Selection_fields.php (hunk 2 above)
Comment (vetusko) [inline views/default/form_Element_OphInBiometry_Selection_fields.php:6]: 1) No controller defined for this - we keep it in the view for now
2) That is the requirement to be able to save data without selecting a lens option
